admin/helpers/healthcheck.js

```diff
@@ -15,11 +21,16 @@ module.exports = {
         }
       })
     })
+    if (cacheBust || !cachedCommitId || !cachedCommitIdExpiresAt || now > cachedCommitIdExpiresAt) {
+      cachedCommitId = await readCommitIdFromFileAsync
+      cachedCommitIdExpiresAt = Date.now() + (60 * 1000) // +60 seconds
```

Comment:
How does the commit ever change?  Sure it can only change on new deployment? If so, why bother with cache-busting?  Perhaps I am missing something?